onnxruntime/core/session/provider_registration.cc

```diff
@@ -12,6 +12,12 @@
 #include "core/session/ort_apis.h"
 #include "core/providers/openvino/openvino_provider_factory_creator.h"
 
+#ifdef _WIN32
+#include <Windows.h>  // For some reason, including tracing.h is having compile errors without this
```

Comment:
You removed "#include <Windows.h>" from "core/platform/tracing.h".  Should it be added back?